<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Acceptfriend</class>
 <widget class="QWidget" name="Acceptfriend">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>551</width>
    <height>301</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QPushButton" name="acceptButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>260</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;幼圆&quot;;</string>
   </property>
   <property name="text">
    <string>接受</string>
   </property>
  </widget>
  <widget class="QPushButton" name="denyButton">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>260</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;幼圆&quot;;</string>
   </property>
   <property name="text">
    <string>拒绝</string>
   </property>
  </widget>
  <widget class="QLabel" name="head">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>91</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-width:0px;
border-radius:40px;
border-image: url(:/src/GUI/head/Genshin/genshin1.jpg);
border-style:solid;
border-color: rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>:/image/toolicon/pokemon-trainer.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="msg">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>166</y>
     <width>341</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none;
</string>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;申请添加您为好友。&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QWidget" name="title" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>551</width>
     <height>55</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(36, 200, 254);</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>41</width>
        <height>41</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>41</width>
        <height>41</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/src/GUI/icon/addFriend.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>:/image/toolicon/ball.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Roboto</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(112, 112, 112);
border:none;</string>
      </property>
      <property name="text">
       <string>  好友申请</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="usernameLabel">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>130</y>
     <width>351</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none;
</string>
   </property>
   <property name="text">
    <string>用户xx（123456）</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>551</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <spacer name="horizontalSpacer">
      <property name="font">
       <font>
        <family>Roboto</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>9</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="closeButton">
      <property name="minimumSize">
       <size>
        <width>16</width>
        <height>16</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16</width>
        <height>16</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Roboto</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
    image: url(:/src/GUI/close.png);
    border-style: outset;
}
QPushButton:hover{
    image: url(:/src/GUI/close_hover.png);
    border-style: outset;
}
QPushButton:pressed{
    image: url(:/src/GUI/close_hover.png);
    border-style: outset;
}
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="minimizeButton">
      <property name="minimumSize">
       <size>
        <width>16</width>
        <height>16</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16</width>
        <height>16</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Roboto</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
    image: url(:/src/GUI/min.png);
    border-style: outset;
        border: none;
}
QPushButton:hover{
    image: url(:/src/GUI/min_hover.png);
    border-style: outset;
        border: none;
}
QPushButton:pressed{
    image: url(:/src/GUI/min_hover.png);
    border-style: outset;
        border: none;
}
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="font">
       <font>
        <family>Roboto</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
